services.yaml

```diff
@@ -1753,13 +1753,47 @@ paths:
     post:
       tags:
         - Events
-      summary: Force Event Snapshot
+      summary: Force event snapshot
       description: Force snapshot on the next time the event occurs
       responses:
         200:
           description: OK
         401:
           $ref: "#/definitions/UnauthorizedError"
+  /services/{service_id}/events/{event_id}/actions:
+    get:
+      tags:
+        - Events
+      summary: "Fetch event actions"
+      description: "Fetch event actions"
+      parameters:
+          - in: path
```

Comment:
please make sure your "tab" are 2 characters and not 4

Reply:
fixed